client/db/types.go

```diff
@@ -11,6 +11,9 @@ import (
 	"strings"
 	"time"
 
+	"decred.org/dcrdex/server/account"
+	"github.com/decred/dcrd/crypto/blake256"
+
 	"decred.org/dcrdex/client/asset"
```

Comment:
Please group and sort the non-std library imports together with no newlines.

Reply:
Will do.